docs/editor_manual/new_pages/the_promote_tab.rst

```diff
@@ -7,12 +7,13 @@ A common feature of the *Edit* pages for all page types is the three tabs at the
 The Promote tab
 ~~~~~~~~~~~~~~~
 
-The second, *Promote*, is where you can set all the 'metadata' (data about data!) for the page. Below is a description of all default fields in the promote tab and what they do.
+The promote tab is where you can configure a page's metadata to help search engines find and index it. Below is a description of all default fields in the promote tab and what they do.
 
-* **Slug:** The last part of the web address for the page. E.g. the slug for a blog page called 'The best things on the web' would be the-best-things-on-the-web (``www.example.com/blog/the-best-things-on-the-web``). This is automatically generated from the main page title set in the Content tab. This can be overridden by adding a new slug into the field. Slugs should be entirely lowercase, with words separated by hyphens (-).
-* **Page title:** An optional, search-engine friendly page title. This is the title that appears in the tab of your browser window. It is also the title that would appear in a search engine if the page was returned as part of a set of search results.
+* **Slug:** The section of the URL that appears after your website's domain e.g ``http://domain.com/blog/[my-slug]/``. This is automatically generated from the main page title set in the Content tab. Slugs should be entirely lowercase, with words separated by hyphens (-). It is recommended that you don't change a page's slug once a page is published.
+* **Title tag:** This is the bold headline in search engine results. This is one of the most significant elements of how search engines rank the page. The keywords used here should align with the keywords you wish to be found for.
+* **Meta description:** This is the descriptive text displayed underneath a headline in search engine results. It is designed to explain what this page is about. It has no impact on how Google chooses to rank your content, but it can impact on the likelihood that a user will click your result. Ideally 140 - 155 characters in length.
```

Comment:
```suggestion
* **Meta description:** This is the descriptive text displayed underneath a headline in search engine results. It is designed to explain what this page is about. It has no impact on how Google ranks your content, but it can impact on the likelihood that a user will click your result. Ideally 140 to 155 characters in length.
```